corehq/blobs/management/commands/check_blob_logs.py

```diff
@@ -120,7 +120,7 @@ def check_blob(rec, old_db, new_db, migrate=False):
 
     if old_db.exists(key=key):
         if migrate:
-            with old_db.get(key=key) as content:
+            with old_db.get(key=key, type_code=CODES.maybe_compressed) as content:
```

Comment:
I see that `key` comes from `rec["blob_key"]`, and that below, `rec["type_code"]` is available.  That would've been my guess, not being familiar with this code.

It looks like `S3BlobDB.get` uses `type_code` only to make sure form XML isn't accessed by key.  I see `maybe_compressed` is "used to get stored blob bytes (no blobs should have this type)" - is this for certain bulk operations (like this one) where we're only accessing the binary data or something?

Reply:
Yes, exactly. We can't use `rec["type_code"]` since that would still fail for form's which may or may not be compressed and require you to pass the full `meta` object. If we passed the full meta object we would then get the uncompressed data which would then get copied as is to the other blob DB. What we want it to copy the blob as it is in the source DB.

The `maybe_compressed` type code is used to get the raw blob data ignoring any compression.

Reply:
Makes sense, thanks for the explanation.